datafusion/core/tests/sql/functions.rs

```diff
@@ -445,3 +445,169 @@ async fn case_builtin_math_expression() {
         assert_batches_sorted_eq!(expected, &results);
     }
 }
+
+#[tokio::test]
+async fn case_sensitive_identifiers_aggregates() {
```

Comment:
I still think this test is accidentally included in this PR. It currently exists in aggregates.rs: 

https://github.com/apache/arrow-datafusion/blob/cc496f87caf8d13e4c17025e0d3b86643a21a243/datafusion/core/tests/sql/aggregates.rs#L1112

Reply:
Fixed, probably a merge issue. Tried to be in sync with fast changing files